ugvc/mrd/featuremap_utils.py

```diff
@@ -43,6 +43,12 @@ class FeatureMapFields(Enum):
     X_CIGAR = "X_CIGAR"
     PREV_3bp = "prev_3bp"
     NEXT_3bp = "next_3bp"
+    PREV_1 = "next_1sbp"
+    PREV_2 = "next_2sbp"
+    PREV_3 = "next_3sbp"
+    NEXT_1 = "prev_1sbp"
```

Comment:
NEXT_1 should be "next_1" not "prev_1"

Reply:
wow, sorry - my mistake, happy you noticed, fixed